### PR TITLE
support Remote group edit

### PR DESCRIPTION
**Pull request by mariakleiner**
b/244219679

### CODE REVIEW

File: pkg/Library/App/Worker/Arcs.js [hunk 1]
```diff
@@ -89,6 +89,8 @@ const handleServiceCall = async msg => {
   socket.sendVibration({kind: 'serviceResult', sid: msg.sid, data});
 };
 
+// public API
```
Comment (sjmiles) [inline pkg/Library/App/Worker/Arcs.js:92]: This is great! ...`init` should go here too please

File: pkg/Library/User/Profile.js [hunk 2]
```diff
@@ -27,11 +27,16 @@ async onLogoutClick(inputs, state, {service}) {
 },
 
 onPersonaChange({eventlet: {value: persona}}) {
-  log(persona);
+  log(`persona = ${persona}`);
   return {persona};
 },
 
-render({persona, profile}, state) {
+onGroupChange({eventlet: {value: group}}) {
```
Comment (sjmiles) [inline pkg/Library/User/Profile.js:34]: `Profile` is a user login thing. Suggest making a new Particle ... 'LobbySettings' even that specific is ok.

It only needs to have the group label (technically) although any other iconography or whatever is cool.

**Review by sjmiles — CHANGES_REQUESTED** [review]

File: pkg/Library/App/Worker/Arcs.js (hunk 1 above)
Comment (mariakleiner) [reply pkg/Library/App/Worker/Arcs.js:86]: done

File: pkg/Library/User/Profile.js (hunk 2 above)
Comment (mariakleiner) [reply pkg/Library/User/Profile.js:34]: done

**Comment by mariakleiner** [comment]
I addressed comments, but something went poorly with the merge... so I should probably create a new PR from a clean branch...

**Pull request closed without merging**